src/strategies/hats-protocol-hat-ids/README.md

```diff
@@ -0,0 +1,30 @@
+# hats-protocol-hat-id
+
+Grants voting power based on if voter has a set of specified hat (IP based).
+
+Here is an example of parameters:
+
+```json
+{
+  "address": "0x3bc1A0Ad72417f2d411118085256fC53CBdDd137",
+  "hatIds": "[\"68\"]"
+}
+```
+
+or 
+
+```json
+{
+  "address": "0x3bc1A0Ad72417f2d411118085256fC53CBdDd137",
+  "hatId": "[\"68\", \"68.1\"]"
+}
+```
+
+or
+
+```json
+{
+  "address": "0x3bc1A0Ad72417f2d411118085256fC53CBdDd137",
+  "hatId": "[\"68\", \"68.1\", \"68.1.1\"]"
+}
```

Comment:
Looks like you are using `id` instead of `ids`

Reply:
good catch! thank you.